go.mod

```diff
@@ -88,7 +88,7 @@ require (
 	github.com/mrunalp/fileutils v0.0.0-20171103030105-7d4729fb3618 // indirect
 	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822
 	github.com/mvdan/xurls v1.1.0
-	github.com/onsi/ginkgo v1.10.1
+	github.com/onsi/ginkgo v1.10.4-0.20191206104214-f66e896ca772
```

Comment:
any chance we can wait for 1.10.4?

Reply:
oh right....
I've been reading the actual vendor diffs :sweat_smile: 
/hold

Reply:
Sure! No probs.

Reply:
do we need to request a release upstream or?

Reply:
If one is not forthcoming and they don't have a regular release cadence, I guess so?

Reply:
I usually ping the project owner to cut a release whenever I need to bump a dependency. I can do that.

Reply:
I glanced around and didn't see much about releases so 😅

Reply:
/hold cancel